src/components/pages/SideDrawer/SideDrawer.js

```diff
@@ -1,206 +1,94 @@
-import IconButton from '@material-ui/core/IconButton';
-import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
-import ChevronRightIcon from '@material-ui/icons/ChevronRight';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemIcon from '@material-ui/core/ListItemIcon';
-import ListItemText from '@material-ui/core/ListItemText';
-import CloseIcon from '@material-ui/icons/Close';
-import RateReviewIcon from '@material-ui/icons/RateReview';
-import AccountIcon from '@material-ui/icons/AccountCircle';
-import HelpIcon from '@material-ui/icons/Help';
-import BuildIcon from '@material-ui/icons/Build';
-import { Link, withRouter } from 'react-router-dom';
-import { SideDrawerData } from './SideDrawerData';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Menu, Button } from 'antd';
 
-const drawerWidth = 215;
+import {
+  DotChartOutlined,
+  MenuUnfoldOutlined,
+  MenuFoldOutlined,
+  PieChartOutlined,
+  FileDoneOutlined,
+  ToolOutlined,
+  FolderOpenOutlined,
+  HistoryOutlined,
+} from '@ant-design/icons';
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    display: 'flex',
-  },
-  appBar: {
-    zIndex: theme.zIndex.drawer + 1,
-  },
-  drawer: {
-    width: drawerWidth,
-    flexShrink: 0,
-    whiteSpace: 'nowrap',
-  },
-  drawerPaper: {
-    width: drawerWidth,
-    height: '100%',
-    top: 87,
-  },
-  drawerOpen: {
-    width: drawerWidth,
-    transition: theme.transitions.create('width', {
-      easing: theme.transitions.easing.sharp,
-      duration: theme.transitions.duration.enteringScreen,
-    }),
-  },
-  drawerClose: {
-    transition: theme.transitions.create('width', {
-      easing: theme.transitions.easing.sharp,
-      duration: theme.transitions.duration.leavingScreen,
-    }),
-    overflowX: 'hidden',
-    width: theme.spacing(7) + 1,
-    [theme.breakpoints.up('sm')]: {
-      width: theme.spacing(9) + 1,
-    },
-  },
-  toolbar: {
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'flex-end',
-    padding: theme.spacing(0, 1),
-  },
-}));
+// example path to images
+import CaseIcon from '../../../styles/icons/case.svg';
 
-function SideDrawer(props) {
-  const classes = useStyles();
-  const [open, setOpen] = useState(true);
-  const [mobile, setMobile] = useState(window.innerWidth < 769);
-  const {
-    logout,
-    location: { pathname },
-  } = props;
+const { SubMenu } = Menu;
+
+function SideDrawer() {
+  const [collapsed, setCollapsed] = useState(false);
 
   const role = window.localStorage.getItem('role');
 
-  useEffect(() => {
-    window.addEventListener(
-      'resize',
-      () => {
-        const mobile = window.innerWidth < 769;
-        if (!mobile) {
-          setMobile(mobile);
-        }
-      },
-      false
-    );
-    return () => {
-      window.removeEventListener('resize');
-    };
-  }, [mobile]);
+  const toggleCollapsed = () => {
+    setCollapsed(!collapsed);
+  };
```

Comment:
The code you got rid of was for mobile styling. It would allow the page to resize when on a phone screen but this can be accomplished better with some minor tweaks in the less file. Great clean-up!